tests/test_user.py

```diff
@@ -0,0 +1,19 @@
+from unittest import TestCase, mock
+import datetime
+
+import twitter_cleanup
+
+
+class UserTest(TestCase):
+    @mock.patch("twitter_cleanup.user.BotometerResult")
+    @mock.patch("twitter_cleanup.user.datetime")
+    def test_last_status_before(self, dt, bot):
+        dt.now.return_value = datetime.datetime(2019, 12, 2, 0, 0, 0, 0)
+        user = twitter_cleanup.User()
+        self.assertFalse(user.last_status_before())
+
+        user.status = mock.Mock()
+        user.status.created_at = datetime.datetime(2019, 12, 1, 0, 0, 0, 0)
+        self.assertTrue(user.last_status_before())
+
+        self.assertFalse(user.last_status_before(hours=48))
```

Comment:
I would recommend one test case for each assertion here. The test method names would clarify the intention of each assertion and the test suite would be clearer for human readers. But this is up to you, I'm just thinking out loud here imagining something like `test_last_status_before_fails_if_post_older_than_today` hahaha...

Also, if you're refactoring this test case, you might want to take a look in [FreezeGun](https://pypi.org/project/freezegun/) ; )